### PR TITLE
Notebook showing batch job

### PR DESCRIPTION
**Pull request by Pratichhya**
Updated simple batchjob

### CODE REVIEW

**Comment by jdries** [comment]
- there's a 'basics' notebook alongside the actual notebook again
- please execute the batch job with 'execute_batch', it is much shorter and otherwise people start copying the long form which is only needed in special cases
- maybe rely more on the special widgets to render things like results? This is much nicer then printing the dictionaries. 
- For batch jobs, we should certainly also point to the web editor, which is the best way for monitoring and management.

File: openeo/Batch_job.ipynb [hunk 1]
```diff
@@ -0,0 +1,2214 @@
...
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# load your data collection\n",
+    "cube = connection.load_collection(\n",
+    "                            \"SENTINEL2_L2A\",\n",
+    "                            bands = [\"B04\", \"B03\", \"B02\"],\n",
```
Comment (soxofaan) [inline openeo/Batch_job.ipynb:89]: use 4 space indentation here

**Comment by Pratichhya** [comment]
> * there's a 'basics' notebook alongside the actual notebook again
The 'basics' notebook is from the main branch. Removal of this notebook results in github action failure. 

> * please execute the batch job with 'execute_batch', it is much shorter and otherwise people start copying the long form which is only needed in special cases
`execute_batch()` is used instead of `start_and_wait()` method of executing job.

> * maybe rely more on the special widgets to render things like results? This is much nicer then printing the dictionaries.
adotped the method suggested by @soxofaan 

> * For batch jobs, we should certainly also point to the web editor, which is the best way for monitoring and management.
Pointed to the web editor in the description section

File: openeo/Batch_job.ipynb (hunk 1 above)
Comment (Pratichhya) [reply openeo/Batch_job.ipynb]: updated

**Review by soxofaan — APPROVED** [review]

**Pull request merged**